### PR TITLE
Block reapplying and fixed CSS for status

### PR DESCRIPTION
**Pull request by mario-elshaer**
The user is now unalloyed to reapply to the same position. And can now see his status on the posting if he decided to try and reapply

### CODE REVIEW

File: client/src/components/Postings/Posting.js [hunk 1]
```diff
@@ -287,10 +317,29 @@ const Posting = () => {
...
+      } else if (application && application.hasOwnProperty("applications")) {
+        return (
+          <span
+            className={`applicationStatus ${application.applications[0]?.applicationStatus}`}
+          >
+            {application.applications[0].applicationStatus}
+          </span>
+        );
+      }
+    };
```
Comment (ramy-att) [inline client/src/components/Postings/Posting.js:340]: @mario-elshaer i made some minor changes

File: client/src/components/Postings/Postings.css [hunk 2]
```diff
@@ -95,3 +95,57 @@ th {
...
+
+  background-color: #4CAF50 !important;
+  display: inline-block;
+    padding: 5px 10px;
+    border: 2px solid #4CAF50;
+    border-radius: 20px;
+    text-align: center;
+      line-height: 20px;
+        height: 40px;
+}
```
Comment (ramy-att) [inline client/src/components/Postings/Postings.css:151]: looks good! Small suggestion: you could have used the general class first to do the general styles

**Review by ramy-att — APPROVED** [review]
Left small comment if you want to take a look, not a big deal however so we can merge the changes. Good job!

File: client/src/components/Postings/Posting.js (hunk 1 above)
Comment (mario-elshaer) [reply client/src/components/Postings/Posting.js:340]: All good!

**Pull request merged**